.github/workflows/build-test.yml

```diff
@@ -29,33 +29,56 @@ jobs:
       - name: Restore node_modules
         uses: OffchainLabs/actions/node-modules/restore@main
 
-      - uses: ./.github/actions/create-env-file
+      - name: Create .env file
+        uses: SpicyPizza/create-envfile@v1.3
```

Comment:
why do we create an env file instead of using `env`?
https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

Reply:
By coping from `.env.sample`, we can reuse the defaulted variables in `.env.sample`

Reply:
We didn't really do that, though, we were just creating a fresh file. Since it's just one variable I'll use `env`